### ISSUE TITLE
Lexical or Preprocessor Issue (Xcode): 'home_indicator-Swift.h' file not found

### ISSUE THREAD

**Issue by Danny-Z** [RainwayApp/flutter_home_indicator#13]
兄弟， 感谢发布这个插件， 我用了很久，非常棒， 但是最近升级的项目， 发现编译出现问了， 找不到头文件， 所以我新建了一个插件发现头文件这样式儿定义的

#if __has_include(<home_indicator/home_indicator-Swift.h>)
#import <home_indicator/home_indicator-Swift.h>
#else
// Support project import fallback if the generated compatibility header
// is not copied when this plugin is created as a library.
// https://forums.swift.org/t/swift-static-libraries-dont-copy-generated-objective-c-header/19816
#import "home_indicator-Swift.h"
#endif

**Comment by nocomeback95** [comment]
@Danny-Z can u make a fork repo ?

**Comment by nohli** [comment]
@Danny-Z use this in your pubspec.yaml:

```
  home_indicator:
    git:
      url: https://github.com/nohli/flutter_home_indicator.git
      ref: fix-build
```

**Comment by lynn** [comment]
I just published 2.0.2 which contains @nohli's fix. Thank you @nohli !
